packages/components/psammead-most-read/src/item.jsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { oneOf, shape, string } from 'prop-types';
+import styled from 'styled-components';
+import { getCanon, getPica } from '@bbc/gel-foundations/typography';
+import { C_EBON, C_POSTBOX } from '@bbc/psammead-styles/colours';
+import { scriptPropType } from '@bbc/gel-foundations/prop-types';
+import { GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { GEL_GROUP_3_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import {
+  getSerifLight,
+  getSerifMedium,
+} from '@bbc/psammead-styles/font-styles';
+
+const MostReadWrapper = styled.div`
+  display: flex;
+
+  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-top: 4px;
+  }
+
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
+    margin-top: 6px;
+  }
+`;
+
+export const CountWrapper = styled.p`
+  ${({ script }) => script && getCanon(script)};
+  ${({ service }) => getSerifLight(service)}
+  color: ${C_POSTBOX};
+  margin: 0; /* Reset */
```

Comment:
could we get rid of the commented bit

Reply:
This comment explains that we're resetting any margins that are pre-defined by `GEL` and/or `styled component` similar to what we have on other components, see https://github.com/bbc/psammead/blob/2932750562232090da693abd707a5b39d6ac6da7/packages/components/psammead-headings/src/index.jsx#L17